addons/website_livechat/static/src/components/visitor_banner/visitor_banner.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website_livechat.VisitorBanner" owl="1">
+        <div class="o_VisitorBanner">
+            <div class="o_VisitorBanner_sidebar">
+                <div class="o_VisitorBanner_avatarContainer">
+                    <img class="o_VisitorBanner_avatar rounded-circle" src="/mail/static/src/img/smiley/avatar.jpg" alt="Avatar"/>
+                    <t t-if="visitor.is_connected">
+                        <i class="o_VisitorBanner_onlineStatusIcon fa fa-circle" title="Online" role="img" aria-label="Visitor is online"/>
+                    </t>
+                </div>
+            </div>
+            <div class="o_VisitorBanner_content">
+                <t t-if="visitor.country">
+                    <img class="o_VisitorBanner_country o_country_flag" t-att-src="'base/static/img/country_flags/'+visitor.country.code+'.png'"/>
```

Comment:
absolute url, use t-attf

Reply:
Was not the case in legacy, just "copy pasted" here, so how do you get the absolute url ?

Reply:
Just add `/` in front, if that wasn't the case before this should be bugfixed in earlier versions.

Reply:
Current `src` in the context of web client leads to browser getting `/web/base/static/...` which is not what we want.

Reply:
![image](https://user-images.githubusercontent.com/48204996/94120529-84f97080-fe50-11ea-9dc9-e18778f2b507.png)

Need to be fixed then

Reply:
Image is also missing `alt` attribute, so we don't even have replacement text for the broken image.